src/cljs/harja/views/urakka/pot2/paallystekerros.cljs

```diff
@@ -122,7 +122,11 @@
        :leveys (:perusleveys pot2-yhteiset/gridin-leveydet) :validoi [[:ei-tyhja "Anna arvo"]]}
       {:otsikko "Kok.m. (t)" :nimi :kokonaismassamaara :tyyppi :positiivinen-numero :tasaa :oikea
        :leveys (:perusleveys pot2-yhteiset/gridin-leveydet) :validoi [[:ei-tyhja "Anna arvo"]]}
-      {:otsikko "Pinta-ala (m²)" :nimi :pinta_ala :tyyppi :positiivinen-numero :tasaa :oikea
+      {:otsikko "Pinta-ala (m²)" :nimi :pinta_ala :tyyppi :positiivinen-numero :tasaa :oikea :muokattava? (constantly false)
+       :hae (fn [rivi]
+              (* (:leveys rivi)
```

Comment:
Tuli vain mieleen, että koska jokaiselle riville lasketaan näin frontilla aina tuo pituus, niin sen melkein vois laskee jo bäkkärissä ennenkuin dataa palautetaan ja tallentaa sen johonki :pituus -kenttään.

Reply:
Pituutta lasketaan uudestaan jokaisella napin painalluksella kun TR-osoitetta muokataan. Se arvo kyllä lopulta tallennetaan backariin pituus-sarakkeeseen.